vega_sim/parameter_test/parameter/experiment.py

```diff
@@ -106,14 +107,35 @@ def output_logs(
         file_path = final_folder / FILE_PATTERN.format(
             param_name=parameter_name, param_value=value
         )
+
+        file_path_LOB = final_folder / FILE_PATTERN_LOB.format(
+            param_name=parameter_name, param_value=value
+        )
+
         with open(file_path, "w") as f:
             csv_writer = csv.writer(f, delimiter=",")
-            # Keys should not change throughout, so just take the first
+            # Keys should not change throughout, so just take the first            
             headers = (
                 list(result_list[0][0].keys()) if col_ordering is None else col_ordering
             )
+            if 'Order Book Bid Side' in result_list[0][0]:
+                headers.remove('Order Book Bid Side')
+                headers.remove('Order Book Ask Side')
 
             csv_writer.writerow(["Iteration"] + headers)
             for i, res in enumerate(result_list):
                 for row in res:
                     csv_writer.writerow([i] + [row[c] for c in headers])
+        
+        if 'Order Book Bid Side' in result_list[0][0]:
+            with open(file_path_LOB, "w") as f:
+                csv_writer = csv.writer(f, delimiter=",")
+                # Keys should not change throughout, so just take the first
+                headers = (
+                    ['Time Step', 'Order Book Bid Side', 'Order Book Ask Side']
+                )
+
+                csv_writer.writerow(["Iteration"] + headers)
+                for i, res in enumerate(result_list):
+                    for row in res:
+                        csv_writer.writerow([i] + [row[c] for c in headers])
```

Comment:
I feel like there's probably a cleaner way to do this than checking for the string in the header, having a think about what that might be and will get back to you!

Reply:
So, thinking the best thing here is if you:
- Pull out `file_pattern` to an optional argument which defaults to `FILE_PATTERN`
- Add a flag which defaults to `True` for whether to save the `run_config.json` file
- Add an additional field to `SingleParameterExperiment` which is a list of (named?) tuples containing `(file_pattern, column_list)`
   -  Have the default be `None` and if it's set to that stick to standard behaviour
   - If column_list is `None`, include everything

How does that sound?

Reply:
Looks great to me about the first two points, will add them. Could I ask about the third point -  if we'd like to output LOB in a new file (could be done if we pass a list of (file_pattern, column_list) in `SingleParameterExperiment` ), and we will use the column_list in `output_logs`, then to check for the data in column_list, right?

Reply:
Yeah that's what I'm thinking